bootstrap/README.md

```diff
@@ -50,9 +56,9 @@ If you have plugged in your speaker via the headphone jack, you may experience i
 
 8. Run a recipe
 
-    cd ~/Desktop/tjbot/recipes/intro
```

Comment:
we could make intro not require watson credentails, but that's fine.. we can just run the HW tests.